src/nls/root/strings.js

```diff
@@ -569,6 +569,39 @@ define(function (require, exports, module) {
             WEB_1440_900: "Web",
             WEB_1920_1080: "Web"
         },
+        SEARCH: {
+            PLACEHOLDER: "Type to search",
+            HEADERS: {
+                LAYERS: "Layers",
+                CURRENT_DOCS: "Current Documents",
+                RECENT_DOCS: "Recent Documents"
+            },
+            CATEGORIES: {
+                DOCUMENT: "document",
+                CURRENT: "current",
+                RECENT: "recent",
+
+                LAYER: "layer",
+                ARTBOARD: "artboard",
+                LINKED: "linked",
+                LAYER_KIND: {
+                    0: "any",
```

Comment:
These ones get compared to user input to determine if an option should be included in the dropdown. Since that depends on user input, doesn't that mean it varies by location?